### ISSUE TITLE
Angular 9 support?

### ISSUE THREAD

**Issue by MohsinEngineer** [Agezao/confetti-generator#18]
Is it possible to use this library with Angular using typescript? 

I tried using it using the following method. Am I doing something wrong?

**donate.component.ts**
```
import ConfettiGenerator from "confetti-js";
import { jQuery } from 'jquery';
declare var jQuery: any;
.
@Component({
   ....
})
export class DonateComponent implements OnInit {
  public confetti;
  @ViewChild('confettiBody') confettiBody: ElementRef;

  ngOnInit() {
    // var confettiSettings = { target: 'confettiBody' };    // Method 1
    var confettiSettings = { target: this.confettiBody };       // Method 2
    this.confetti = new ConfettiGenerator(confettiSettings); 
  }
  dropConfetti() {
    console.log('dropping confetti');
    this.confetti.render();
  }
}
```

**donate.component.html**
```
<!-- Successful Donation Modal -->
<div #successfulDonationModal class="modal fade" id="successfulDonationModal" tabindex="-1" role="dialog">
  <div class="modal-dialog modal-dialog-centered" role="document">
    <div class="modal-content">
      <div #confettiBody class="modal-body text-center" id="confettiBody">
        <img class="img-fluid" src="assets/img/26.png" alt="smiley-face">
        <h1 class="pt-4">THANK YOU!!</h1>
        <h1 class="">FOR DONATING</h1>
      </div>
    </div>
  </div>
</div>
```

Have tried direct reference using method 1 and also jquery elemRef via method 2 mentioned in donate.component.ts above, both didn't work. Here is the error stack in both:

```
ERROR TypeError: Cannot read property 'getContext' of null
    at new ConfettiGenerator (index.es.js:52)
    at DonateComponent.ngOnInit (donate.component.ts:28)
    at callHook (core.js:3937)
    at callHooks (core.js:3901)
    at executeInitAndCheckHooks (core.js:3842)
    at refreshView (core.js:11795)
    at refreshDynamicEmbeddedViews (core.js:13142)
    at refreshView (core.js:11800)
    at refreshComponent (core.js:13217)
    at refreshChildComponents (core.js:11508)
defaultErrorLogger @ core.js:5873 ...........
```

Is it even possible to do this from the typescript side or have to include .js file to do this for Angular 9?

**Comment by Agezao** [comment]
Hi @MohsinEngineer, _(edit: I tagged the wrong user, big sorry)_

Can you check if the @0.0.18 version works well for you? I strongly believe that it will.

I've intentionally postponed your PR review, sorry, in order to think of an alternative that was less _angular specific_ _(at least on a *documentation* spectre)_ and could positively affect more users of the lib by this.

Turned out that there where other users with the same problem as you that came out with a solution quicker than me. And the before spoke solution got merged on @0.0.18.

Looking forward to feedback on this; See ya!

**Comment by jer-tx** [comment]
@Agezao Running into the exact same problem, and am on 0.0.18.
